WooCommerce/Classes/ViewRelated/Blaze/BudgetSetting/BlazeBudgetSettingView.swift

```diff
@@ -9,6 +9,7 @@ struct BlazeBudgetSettingView: View {
     @State private var showingDurationSetting = false
     @State private var duration: Double = 0
     @State private var startDate = Date()
+    private var minDayAllowedInPickerSelection = Date.now + 60 * 60 * 24 // Current date + 1 day
```

Comment:
Maybe we can make this as a public static constant and use it from `BlazeCampaignCreationFormViewModel` for defining the initial `startDate` value.

Reply:
I applied the changes in this commit d6d4866474df3113298f8928db2806680390ab9b

I moved it under `BlazeBudgetSettingViewModel`, since it's related just to the Budget view. Plus, this variable is not definiting the start date, but the minimum day allowed in the picker, since the start date could be also different from the minimum day allowed (eg. editing a campaign). 👍